examples/complete/versions.tf

```
@@ -1,18 +1,22 @@
 terraform {
-  required_version = ">= 0.12.0"
+  required_version = "~> 1.5"
```

I feel like this should be `>=` so we're not limiting testers TF version?

```suggestion
  required_version = ">= 1.5"
```

---

@Gowiem I treated with example as a typical root module, but I'm not opposed to this suggestion 👍